doc/manual/plugins.rst

```diff
@@ -46,7 +50,6 @@ Global plugins
    plugins_global/errors
    plugins_global/rc
    plugins_global/wcsmatch
-   plugins_global/changehistory
```

Comment:
I like the idea of having the actual source of the documentation in the plugin docstring.  Can we modify our documentation build to generate the pages for the plugins (by simply extracting the string and writing it to a file)?